README.md

```diff
@@ -1,2 +1,40 @@
-# mbed-os-example-mbed-crypto
+# Running Mbed Crypto examples on Mbed OS
+This repository contains a set of examples demonstrating the compilation and use
+of Mbed Crypto on Mbed OS.
 
+List of examples contained within this repository:
+* Cipher encrypt/decrypt using an AES key in cipher block chain (CBC) mode with no padding using a single block.
+* Cipher encrypt/decrypt using an AES key in cipher block chain (CBC) mode with PKCS7 padding using multiple blocks.
+* Cipher encrypt/decrypt using an AES key in counter (CTR) mode using multiple blocks.
+
+## Prerequisites
+* Install <a href='https://github.com/ARMmbed/mbed-cli#installing-mbed-cli'>Mbed CLI</a>
+
+## Deploy
+The following are the steps required for deployment:
+* Clone this repository: `git clone git@github.com:ARMmbed/mbed-os-example-mbed-crypto.git`
```

Comment:
Does `mbed import` work with our example repo? I think it would clone and deploy all in one go. No need to worry about users having git keys set up or anything.

Reply:
I think it would work, but I cannot validate this before the branch is merged because there's no way of specifying a branch with `mbed import`.

Reply:
i have tried `mbed import git clone git@github.com:ARMmbed/mbed-os-example-mbed-crypto`
It works

Reply:
Yes I also validated that it works, the readme will be updated via https://github.com/ARMmbed/mbed-os-example-mbed-crypto/pull/3